notebooks/hf_adapted_llama_inference.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
when loading the HF model and converting to FMS, we should have half types since the beginning, let's avoid people complaining about their gpu getting an OOM that is avoidable

Reply:
we can even use the device map in `from_pretrained` to load directly to GPU